scripts/spack_packages/umpire/package.py

```diff
@@ -229,7 +229,7 @@ def initconfig_mpi_entries(self):
         spec = self.spec
 
         entries = super(Umpire, self).initconfig_mpi_entries()
-        entries.append(cmake_cache_option("ENABLE_MPI", '+mpi' in spec))
+        entries.append(cmake_cache_option("UMPIRE_ENABLE_MPI", '+mpi' in spec))
```

Comment:
I think this needs to remain `ENABLE_MPI`

Reply:
Thank you for the careful review.  I was wondering about this actually.

Reply:
Although, aren't we expecting `cmake` users to specify `-DUMPIRE_ENABLE_MPI=On`?  I'm not sure I follow

Reply:
What is the intended difference between `ENABLE_MPI` and `UMPIRE_ENABLE_MPI`?